api/src/main/java/org/openmrs/module/appointmentscheduling/api/impl/AppointmentServiceImpl.java

```diff
@@ -1155,8 +1159,87 @@ public Appointment bookAppointment(Appointment appointment,
 				appointment);
 	}
 
+	@Override
+	public List<AppointmentDailyCount> getAppointmentsDailyCount(Date fromDate, Date toDate, Location location, Provider provider,
+																 AppointmentType appointmentType, AppointmentStatus status, VisitType visitType) {
+		List<Appointment> appointments = getAppointmentsByConstraints(fromDate,
+				toDate, location, provider, appointmentType, null, Collections.singletonList(status),
+				visitType, null);
+		try {
+			return getAppointmentsDailyReport(appointments);
+		} catch (ParseException e) {
+			e.printStackTrace();
```

Comment:
Why are you catching this exception and handling it in this way.  You should throw a proper exception here.